inhibit_libc.patch

From: Christian Prochaska <christian.prochaska@genode-labs.com>

Make sure to disable the 'inhibit_libc' flag, which is evaluated when
compiling libgcc. When libc is inhibited, the 'unwind-dw2-fde-glibc.c'
is not using the "new" glibc exception handling mechanism. However,
Genode's dynamic linker relies on this mechanism.
---
 gcc/configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/configure.ac b/gcc/configure.ac
index 29ac2f3..67db551 100644
--- a/gcc/configure.ac
+++ b/gcc/configure.ac
@@ -1888,7 +1888,7 @@ fi
 if { { test x$host != x$target && test "x$with_sysroot" = x ; } ||
        test x$with_newlib = xyes ; } &&
      { test "x$with_headers" = x || test "x$with_headers" = xno ; } ; then
-       inhibit_libc=true
+       inhibit_libc=false # was true
 fi
 AC_SUBST(inhibit_libc)
 
